tests/assertions.js

```diff
@@ -153,3 +153,56 @@ function _shouldHaveClaims({subject}) {
     'Expected credentialSubject to have at least one claim.'
   );
 }
+
+export function shouldBeSecured(name, issuedVc) {
+  if('proof' in issuedVc) {
```

Comment:
`issuedVc` is showing up as `undefined` as me (sometimes) and causing these tests to fail, so...needs more work, sadly.

Reply:
@PatStLouis problem here is that the `issuedVc` isn't quite ready for the first test as `before()` isn't awaited before the first/any `it()` statements kick off.

Reply:
a before with an async in front of a it should be awaited, problem is probably that the endpoint is erroring and the try catch is just looking the error. You do need to check here for `expect(issuedVc, `${vendorName} failed to issue error`).to.exist`